feat(issues): Add more `is:for_review` details (#11023)
getsentry/sentry-docs: docs/concepts/search/searchable-properties/issues.mdx

### `is`

> The properties of an issue. Values can be: `unresolved`, `resolved`, `archived`, `assigned`, `unassigned`, `for_review`, `linked`, or `unlinked`. The `for_review` value filters unresolved issues to just the recently unresolved or new issues that have not yet been marked reviewed.  The `linked` and `unlinked` values return issues based on whether they are linked to an external issue tracker or not.

Reviewer: ```suggestion
The properties of an issue. Values can be: `unresolved`, `resolved`, `archived`, `assigned`, `unassigned`, `for_review`, `linked`, or `unlinked`. The `for_review` value filters unresolved issues and only shows the recently unresolved or new issues that haven't been marked as "reviewed".  The `linked` and `unlinked` values return issues based on whether they're linked to an external issue tracker or not.
```
